### PR TITLE
Release Akka 2.6.1

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: blog/_posts/2019-12-06-akka-2.6.1-released.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+For this release we had the help of 26 committers – thank you all very much!
+
+```
+commits  added  removed
+     21   1627      608 Patrik Nordwall
+     19   1211      764 Arnout Engelen
+     18     23       23 Robert Stoll
+     16   1001     1011 Johan Andrén
+     16     20       23 Scala Steward
```
Comment (johanandren) [inline blog/_posts/2019-12-06-akka-2.6.1-released.md]: Skip steward?

File: blog/_posts/2019-12-06-akka-2.6.1-released.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+Dear hakkers,
+
+We are pleased to announce a new patch release of Akka 2.6. Notable changes relative to 2.6.0 include:
+
+ * Supporting Jackson serializers without outputting a manifest [#28113](https://github.com/akka/akka/issues/28113)
+ * Many documentation updates, thanks to [Robert Stoll](https://github.com/robstoll) and others
+ * Improved `ByteString.toArray` performance under Scala 2.13 [#28114](https://github.com/akka/akka/issues/28114)
+ * Lifecycle events for Artery [#28003](https://github.com/akka/akka/issues/28003)
+ * Structured log events [#28207](https://github.com/akka/akka/issues/28207)
+ * 'localAffinity' for group routers [#28174](https://github.com/akka/akka/issues/28174)
```
Comment (johanandren) [inline blog/_posts/2019-12-06-akka-2.6.1-released.md]: Perhaps a thanks to pxsdirac here

Comment (raboof) [reply blog/_posts/2019-12-06-akka-2.6.1-released.md]: good one I'll add credits for all of them

File: blog/_posts/2019-12-06-akka-2.6.1-released.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+---
+
+Dear hakkers,
+
+We are pleased to announce a new patch release of Akka 2.6. Notable changes relative to 2.6.0 include:
+
+ * Supporting Jackson serializers without outputting a manifest [#28113](https://github.com/akka/akka/issues/28113)
+ * Many documentation updates, thanks to [Robert Stoll](https://github.com/robstoll) and others
+ * Improved `ByteString.toArray` performance under Scala 2.13 [#28114](https://github.com/akka/akka/issues/28114)
+ * Lifecycle events for Artery [#28003](https://github.com/akka/akka/issues/28003)
```
Comment (patriknw) [inline blog/_posts/2019-12-06-akka-2.6.1-released.md]: I think we can leave this out. It's kind of included in the next item.

File: blog/_posts/2019-12-06-akka-2.6.1-released.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+ * Structured log events [#28207](https://github.com/akka/akka/issues/28207)
+ * 'localAffinity' for group routers [#28174](https://github.com/akka/akka/issues/28174)
+ * A `Flow.delayWith` that allows a custom delay for each element [#25000](https://github.com/akka/akka/pull/25000)
+ * Loosen `cluster.role.<role-name>.min-nr-of-members` fallback [#28177](https://github.com/akka/akka/issues/28177)
+
+A total of 77 issues were closed since 2.6.0. The complete list can be found on the [2.6.1 milestone](https://github.com/akka/akka/milestone/155?closed=1) on github.
```
Comment (patriknw) [inline blog/_posts/2019-12-06-akka-2.6.1-released.md]: wrong link, it should be https://github.com/akka/akka/milestone/156?closed=1

File: blog/_posts/2019-12-06-akka-2.6.1-released.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+We are pleased to announce a new patch release of Akka 2.6. Notable changes relative to 2.6.0 include:
+
+ * Supporting Jackson serializers without outputting a manifest [#28113](https://github.com/akka/akka/issues/28113)
+ * Many documentation updates, thanks to [Robert Stoll](https://github.com/robstoll) and others
+ * Improved `ByteString.toArray` performance under Scala 2.13 [#28114](https://github.com/akka/akka/issues/28114)
+ * Lifecycle events for Artery [#28003](https://github.com/akka/akka/issues/28003)
+ * Structured log events [#28207](https://github.com/akka/akka/issues/28207)
+ * 'localAffinity' for group routers [#28174](https://github.com/akka/akka/issues/28174)
+ * A `Flow.delayWith` that allows a custom delay for each element [#25000](https://github.com/akka/akka/pull/25000)
+ * Loosen `cluster.role.<role-name>.min-nr-of-members` fallback [#28177](https://github.com/akka/akka/issues/28177)
```
Comment (patriknw) [inline blog/_posts/2019-12-06-akka-2.6.1-released.md]: a few more candidates
* https://github.com/akka/akka/pull/27875
* consistent hashing router https://github.com/akka/akka/issues/27729 thanks to navaro1
* fix supervision for Typed ClusterSharding and ClusterSingleton https://github.com/akka/akka/issues/28035

File: blog/_posts/2019-12-06-akka-2.6.1-released.md (hunk 2 above)
Comment (patriknw) [inline blog/_posts/2019-12-06-akka-2.6.1-released.md]: thanks to pxsdirac

File: blog/_posts/2019-12-06-akka-2.6.1-released.md (hunk 1 above)
Comment (patriknw) [inline blog/_posts/2019-12-06-akka-2.6.1-released.md]: did we decide to always exclude steward from this? maybe fix in script?

File: blog/_posts/2019-12-06-akka-2.6.1-released.md (hunk 5 above)
Comment (patriknw) [reply blog/_posts/2019-12-06-akka-2.6.1-released.md]: and the extensions is important https://github.com/akka/akka/pull/28294
please update the milestone for that PR

**Pull request merged**